internal/provider/schema_gitlab_project_hook.go

```diff
@@ -0,0 +1,143 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func gitlabProjectHookSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"project": {
+			Description: "The name or id of the project to add the hook to.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"project_id": {
+			Description: "The id of the project for the hook.",
+			Type:        schema.TypeInt,
+			Computed:    true,
+		},
```

Comment:
@timofurrer  - We seem to have gained `project_id` as part of migrating from the resource -> schema. If this just an attempt to always have the numeric value available for downstream resources as a result of something like #1165 ?

Reply:
Yes, exactly. And it's actually returned as `project_id` by the upstream API, too.